.github/pull_request_template.md

```diff
@@ -1,14 +1,9 @@
 ## Overview
 
-Brief description of what this PR does, and why it is needed.
-
-### Demo
-
-Optional. Screenshots, `curl` examples, etc.
+Connects #XXX
 
-### Notes
+Brief description of what this PR does, and why it is needed.
```

Comment:
I'm confused why these changes are being displayed in the diff, didn't you already pull them in in #117?

Reply:
I just had to pull in from master!